### DISCUSSION TITLE
Balance incorrect after deleting asset account

### DISCUSSION THREAD

**Discussion by chivagio** [firefly-iii/firefly-iii#8887] [Firefly III] [answered]
Hello,
I've just deleted one asset that I was no longer using and then my main asset's balance increased significantly. In the past there were some transactions between those two assets. I've read the documentation and I know that this is the expected outcome, but is there a way to fix this? 
I've tried looking in the transactions but I can't find anything to delete and get back to the correct balance. 

Any ideas? :'(

**Comment by JC5** [comment, marked as answer]
Sorry, there's not much that can be done to fix that easily. You could make a single transaction that corrects the balance?

**Reply by chivagio** [reply]
Thanks for your quick respone. I've finally decided to reimport the deleted transactions (I've filtered my .CSV file manually to only import what was missing) and now everything is in order.
